.github/workflows/AlwaysOnKotlin.yml

```diff
@@ -58,11 +58,6 @@ jobs:
     needs: build
     runs-on: macOS-latest # enables hardware acceleration in the virtual machine
     timeout-minutes: 30
-    strategy:
-      fail-fast: false
-      matrix:
-        profile: [ wear_round, wear_square ]
-        api-level: [ 26, 28, 30 ]
```

Comment:
Losing the matrix here is unfortunate as the entire job will pass or fail? Can you retain the matrix with the device names?  Will be easy to check which devices are failing, and possibly faster?

Reply:
Good idea! (having them done in one job will use only a single VM instead of 6, but that's probably fine)
 
 All of them are blocked on https://issuetracker.google.com/issues/204590177 right now.